dune-project

```diff
@@ -1,2 +1,77 @@
 (lang dune 2.3)
+
 (name unison)
+
+(version dev)
+
+(generate_opam_files false)
+
+(license GPL-3.0-or-later)
+
+(maintainers "maintainers")
+
+(authors "Trevor Jim, Benjamin C. Pierce and J\195\169r\195\180me Vouillon")
+
+(source
+ (github bcpierce00/unison))
+
+(homepage https://www.cis.upenn.edu/~bcpierce/unison/)
+
+(documentation "https://github.com/bcpierce00/unison/wiki")
+
+(package
+ (name unison)
+ (synopsis "File-synchronization tool for Unix and Windows")
+ (description "
+Text based userinterface for Unison.
+
+Unison is a file-synchronization tool for Unix and Windows.
+It allows two replicas of a collection of files and directories
+to be stored on different hosts (or different disks on the same host),
+modified separately, and then brought up to date by propagating
+the changes in each replica to the other.
+")
+ (depends
+  (ocaml
+   (>= 4.03))
+  (dune
+   (>= 2.3))))
+
+(package
+ (name unison-gui)
+ (synopsis "File-synchronization tool for Unix and Windows")
+ (description "
+Graphical userinterface for Unison.
+
+Unison is a file-synchronization tool for Unix and Windows.
+It allows two replicas of a collection of files and directories
+to be stored on different hosts (or different disks on the same host),
+modified separately, and then brought up to date by propagating
+the changes in each replica to the other.
+")
+ (depends
+  (ocaml
+   (>= 4.03))
+  (dune
+   (>= 2.3))
+  (lablgtk2
+   (>= 2.18.6))))
+
+(package
+ (name unison-fsmonitor)
+ (synopsis "File-synchronization tool for Unix and Windows")
+ (description "
+This helper application monitors file system changes on all given
+(relative to root) paths.
```

Comment:
I don't have a good wording suggestion right now but I think it should be made even more clear that this is an _optional add-on_ to the main application. Perhaps by adding that without this helper the syncing, while fully functional, is done manually, repeated with a set time interval, or initiated externally (cron, for example).